src/mlpack/bindings/R/print_doc_functions_impl.hpp

```diff
@@ -153,14 +217,89 @@ std::string ProgramCall(const std::string& programName, Args... args)
   oss.str(""); // Reset it.
 
   // Now process each output option.
-  oss << PrintOutputOptions(args...);
+  oss << PrintOutputOptions(markdown, args...);
+  if (markdown)
+  {
+    if (oss.str() == "")
+      return util::HyphenateString(call, 2);
+    else
+      return util::HyphenateString(call, 2) + "\n" + oss.str();
+  }
+  
   if (oss.str() == "")
     return "\\donttest{\n" + util::HyphenateString(call, 2) + "\n}";
   else
     return "\\donttest{\n" + util::HyphenateString(call, 2) + "\n" + oss.str() +
            "\n}";
 }
 
+/**
+ * Given the name of a binding, print a program call assuming that all options
+ * are specified.  The programName should not be the output of GetBindingName().
+ */
+inline std::string ProgramCall(const std::string& programName)
+{
+  std::ostringstream oss;
+  oss << "R> ";
```

Comment:
Maybe set this as a command line prefix default variable?

Reply:
This function use to build markdown documentation for our website(For e.g. : https://www.mlpack.org/doc/mlpack-3.3.2/julia_documentation.html#adaboost ). This will be not used in our R package so I thought it may be a nice way to represent it , but I can change it as a command line prefix.

Reply:
The prompt for an R workspace is just `>`, so personally I think that could be okay to use here.

Reply:
(It's configurable; and some of us use `R>` do make it more visually distinct. But either form works.)

Reply:
@Yashwants19 sorry for the late reply on this one. But, when I said:

> Maybe set this as a command line prefix default variable?

My thought was: 

```cpp
std::string command_prefix = "R> ";
```

As the prefix is listed in two places within the file.

Reply:
@coatless I am really sorry, I misunderstood this part, I will update as suggested.

Reply:
I have updated this in the last commit.

Reply:
Awesome! Sorry for the initial feedback not being concise.